TSC1/README.md

```diff
@@ -26,7 +26,7 @@ The TSC engages in various sorts of communication with different audiences. Is i
 | UberConference | [Conference Room](https://www.uberconference.com/opendatakit) | Anyone can join, previous calls are not visible | For meetings and synchronous communication. |
 | Forums | [**tsc** Tag](https://forum.opendatakit.org/tags/tsc) | Usually public | Used for public posts with TSC-related content, usually in the [Governance](https://forum.opendatakit.org/c/governance) category but not always. |
```

Comment:
doesn't look like this tag is used?
<img width="353" alt="Screen Shot 2019-04-15 at 10 44 03 AM" src="https://user-images.githubusercontent.com/4806884/56141772-76c0d880-5f6b-11e9-84c3-140cd1a707b7.png">